buildenv/jenkins/jobs/pipelines/Pipeline_Template.groovy

```diff
@@ -97,6 +97,7 @@ pipelineJob("$JOB_NAME") {
         stringParam('CUSTOM_DESCRIPTION')
         stringParam('SCM_BRANCH', SCM_BRANCH)
         stringParam('SCM_REFSPEC', SCM_REFSPEC)
+        booleanParam('ARCHIVE_JAVADOC')
```

Comment:
Does this need a default value?
https://jenkinsci.github.io/job-dsl-plugin/#path/pipelineJob-parameters-booleanParam

Reply:
I think we also need a default value here. Thanks for the link!